### PR TITLE
[#163054673]Re-implement single meetup page

### PR DESCRIPTION
**Pull request by NedyUdombat**
#### What does this PR do?
This PR:
* Re-implements the single meetup page.

#### Description of Task to be completed
* Remove time text.
* Add text to tell the user about the question section.
* Add image to the single meetup card.
* change the question icon to a plus icon on the ask question  button
* Move question form into a modal
* make upvote icon to have blue color
* make downvote icon to have red color
* add padding to the ask question form modal
* align the address text

#### How should this be manually tested?
visit https://nedyudombat.github.io/Questioner/UI/single-meetup.html

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?(if applicable)
 #163038938

### CODE REVIEW

File: UI/assets/js/main.js [hunk 1]
```diff
@@ -1,111 +1,94 @@
...
-}
-downVoteBtn.onclick = function () {
-  if (upVoteCount%2 !== 0) {
-    downVote();
-  } else {
-    return
-  }
-  downVoteCount ++;
-}
+// // upvote funtion
```
Comment (robocopkaka) [inline UI/assets/js/main.js]: Why was this function commented out?

Comment (NedyUdombat) [reply UI/assets/js/main.js]: This was an oversight, while trying to refactor the rsvp function to ES6, it will be un-commented in the next PR.

Comment (robocopkaka) [reply UI/assets/js/main.js]: You should be able to do that in this PR. Pushing a new commit will reflect on this PR

Comment (NedyUdombat) [reply UI/assets/js/main.js]: Okay, I will be doing that right away

Comment (NedyUdombat) [reply UI/assets/js/main.js]: It has been updated in new commit

**Pull request merged**